src/test/scala/uk/gov/hmrc/test/ui/driver/BrowserDriver.scala

```diff
@@ -30,17 +30,20 @@ trait BrowserDriver extends LazyLogging {
   )
 
   val browser: Option[String] = sys.props.get("browser")
-  if (browser.isEmpty)
+  if (browser.isEmpty) {
     sys.props += ("browser" -> "chrome")
+  }
 
   private val options = browser match {
-    case Some("chrome") | None => Some(new ChromeOptions().addArguments("--remote-allow-origins=*"))
+    case Some("chrome") | None => Some(new ChromeOptions())
```

Comment:
Spoke to Michael, he said it can be removed and asked to remove arguments as it's not needed anymore. 
Good spot :)